scripts/us_census/acs5yr/subject_tables/common/data_loader.py

```diff
@@ -367,19 +371,20 @@ def _generate_mcf_from_column_map(self):
 
         #get cleaned column map
         final_mcf = ""
-
+        # NOTE: this code is duplicated in generate_col_map, any changes here should also be copied there
         for column, stat_var in self.mcf_dict.items():
             col_stat_var = []
             # TODO: Add 'Node:' as the first element of col_stat_var, and append
             # all the stat-var node information to col_stat_var
-            dcid = 'Node: ' + column + '\n'
+            dcid = 'Node: ' + column
+            col_stat_var.append(dcid)
 
             for p, v in stat_var.items():
-                if p != 'unit' or p != 'scalingFactor':
+                if p != 'unit' and p != 'scalingFactor' and p != 'Node':
                     col_stat_var.append(f"{p}: {v}")
             # TODO: Replace the next two lines with final_mcf = final_mcf + '\n'.join(col_stat_var) + '\n\n'
-            col_stat_var = dcid + '\n'.join(col_stat_var)
-            final_mcf = final_mcf + col_stat_var + "\n\n"
+            # col_stat_var = dcid + '\n'.join(col_stat_var)
+            final_mcf = final_mcf + '\n'.join(col_stat_var) + "\n\n"
         return final_mcf
```

Comment:
Having the same function repeated in two places, is a bit difficult to maintain, perhaps, we can move this function to a common module?

Or better is to move this function to `process.py` since the column map is returned as a dictionary to process.py from `common/generate_col_map.py` module